### ISSUE TITLE
[Narrator] When expand and collapse one property's dropdown list to another property, Narrator remembers to read the property that was first entered into the propertyGrid with Alphabetical setting

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#12523]
### .NET version

10.0.100-alpha.1.24570.15

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Cannot repro in .NET 6.0, but repro in .NET 7.0/8.0/9.0

### Issue description

When expand and collapse one property's dropdown list to another property, Narrator remembers to read the property that was first entered into the propertyGrid with Alphabetical setting

**Actual result:**
When setting Alphabetical, go to the propertyGrid where the first property is Button's Text, then expand and close the AutoSize dropdown list, then focus on the AutoEllipsis property.
![Image](https://github.com/user-attachments/assets/7b1be361-32a5-479c-b262-c305146a515d)

https://github.com/user-attachments/assets/cb123c72-59f5-4f70-9954-c1a96d268116

**Expected result:**
When focus on AutoEllipsis property, Narrator doesn't announce the extra Button's Text property 
![Image](https://github.com/user-attachments/assets/52f9cd3a-cbb2-40a1-bcb1-f2550570641d)

https://github.com/user-attachments/assets/2f08fbdb-34ca-4dee-a6f2-8165e2b11c38

### Steps to reproduce

Repro steps:
1.	Create a WinForms application with PropertyGrid bind to Button
2.	Build and run app
3.	Use narrator for testing PropertyGrid, click Alphabetical toolstripbutton firstly - it will focus on Text property
4.	Find one property: AutoSize with dorpdownlist, click it, Alt+Down to expand, Alt+Down to collapse it
5.	Press Up keyboard to another AutoEllipsis property

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang  - I couldn't reproduce this issue. And I didn't see "Text, edit, button1" narrated in your video, only in the screenshot. In video you are navigating with the mouse, I tried both mouse and keyboard. Please see what is missing in my steps:
1. Create a WinForms application with PropertyGrid bind to Button
2. Build and run app
3. start narrator
4. mouse click on the button1 in form1
5. mouse click on the alphabetical sort radio tool strip button -> Text property is focused and announced
7 mouse middle wheel-scroll up to AutoSize, left click on AutoSize,  Alt+Down to expand and Alt+Down to collapse
8 mouse left-click on AutoEllipsis - AutoEllipsis gtets announced as expected, Text property is not mentioned....

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw  - the scenario does not seem a common one, what are your thoughts about the priority?

**Comment by merriemcgaw** [comment]
This would be a usability issue in my mind where there's extra content being given to the user. I'm closing as low priority, but if we get feedback on it this is something we could address.
